packages/nextjs/src/server/errors.ts

```diff
@@ -97,3 +97,5 @@ For additional information about middleware, please visit https://clerk.com/docs
 };
 
 export const authSignatureInvalid = `Clerk: Unable to verify request, this usually means the Clerk middleware did not run. Ensure Clerk's middleware is properly integrated and matches the current route. For more information, see: https://clerk.com/docs/nextjs/middleware. (code=auth_signature_invalid)`;
+
+export const signingKeyInvalid = `Clerk: Unable to decrypt request data, this usually means the signing key is invalid. Ensure the signing key is properly set. (code=signing_key_invalid)`;
```

Comment:
Shall we include a short example or instruct the users to check whether the `CLERK_ENCRYPTION_KEY` env var is defined?

Reply:
+1, we should link to the docs where mention regarding the use case of this key.